microsoft-edge/devtools-guide-chromium/network/reference.md

```diff
@@ -446,149 +620,260 @@ The Requests table displays the following columns by default:
 
-The Waterfall column of the **Requests** pane:
+The **Waterfall** column of the **Requests** pane:
 
 ![The Waterfall column of the Requests pane](./reference-images/requests-waterfall.png)
 
-<!-- ### Analyze the frames of a WebSocket Connection  -->
 
-<!--To view the frames of a WebSocket connection:
+<!-- ------------------------------ -->
+#### Analyze the messages of a WebSocket connection
+<!-- https://developer.chrome.com/docs/devtools/network/reference/#frames -->
 
-1. Click the URL of the WebSocket connection, under the **Name** column of the Requests table.
-1. Click the **Frames** panel.  The table shows the last 100 frames.
+To view the messages of a WebSocket connection:
 
-To refresh the table, re-select the name of the WebSocket connection under the **Name** column of the Requests table.  -->
+1. Under the **Name** column of the **Requests** table, click the URL of the WebSocket connection.
 
-<!--
-![The Frames panel](../media/network-frames.png)
--->
+1. Click the **Messages** tab.  The table shows the last 100 messages.
 
-<!--The table contains the following three columns.
+To refresh the table, re-click the name of the WebSocket connection under the **Name** column of the **Requests** table.
+
+<!-- ![The Frames panel](todo png: network-frames.png) -->
+
+The table contains the following three columns:
+
+*  **Data**.  The message payload.  If the message is plain text, it is displayed here.  For binary opcodes, this column displays the name and code of the opcode.  The following opcodes are supported: 
+   * Continuation Frame
+   * Binary Frame
+   * Connection Close Frame
+   * Ping Frame
+   * Pong Frame
 
-*  **Data**.  The message payload.  If the message is plain text, it is displayed here.  For binary opcodes, this column displays the name and code of the opcode.  The following opcodes are supported: Continuation Frame, Binary Frame, Connection Close Frame, Ping Frame, and Pong Frame.
 *  **Length**.  The length of the message payload, in bytes.
-*  **Time**.  The time when the message was received or sent.  -->
 
-<!--Messages are color-coded according to each type.
+*  **Time**.  The time when the message was received or sent.
+
+Messages are color-coded according to each type:
 
 *  Outgoing text messages are light-green.
 *  Incoming text messages are white.
 *  WebSocket opcodes are light-yellow.
-*  Errors are light-red.  -->
+*  Errors are light-red.
+
+
+<!-- ------------------------------ -->
+#### Analyze events in a stream
+<!-- https://developer.chrome.com/docs/devtools/network/reference/#event-stream -->
+<!-- todo: clean up text/links -->
+
+To view the events that servers stream through [Fetch API](https://web.dev/articles/introduction-to-fetch),<!-- todo: link to MDN instead --> [EventSource API](https://web.dev/articles/eventsource-basics#event_stream_format),<!-- todo: link to MDN instead --> and XHR:
```

Comment:
For server-side events/EventSource API, let's use this MDN page: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events